src/pages/PagesRouter.tsx

```diff
@@ -31,6 +32,7 @@ export const PagesRouter: React.FC = () => {
           <Route path='/staking' component={StakingPage} />
           <Route path={'/exchange'} component={ExchangePage} />
           <Route path={'/stats'} component={StatisticsPage} />
+          <Route path={'/swap'} component={SwapPage} />
```

Comment:
Remove this page for now, storybook should be enough; we will add it after connecting everything to Redux